Automattic/woocommerce-payments
# Fall back to credit card when a payment method is toggled off

Fixes https://github.com/Automattic/woocommerce-payments/issues/8566

#### Changes proposed in this Pull Request
This PR adds the fallback to credit card payment method on the shortcode checkout page to make sure that a new payment method is selected after the currently selected payment method is toggled off because of country location restrictions.









#### Testing instructions





* Enable Giropay, ensure the store currency is EUR
* Navigate to shortcode checkout
* Select Germany in the billing country field, choose Giropay payment method
* Change country to e.g. Belgium
* Giropay now disappears but the checkout page falls back to the credit card payment method by opening it


-------------------


**Post merge**
